src/components/Form/FormTypes.js

```diff
@@ -0,0 +1,6 @@
+const formTypes = {
```

Comment:
Since this is tightly coupled with the form component, it should go in the same file.